src/js/components/getWeather.js

```diff
@@ -0,0 +1,28 @@
+import Time from '../helpers/convertTime';
+import * as el from './elements';
+
+const getWeatherData = async (city) => {
+  const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${el.key}&units=metric`;
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+};
```

Comment:
Hello @DcRonan,
Job well-done.

**Kindly look into error handling as the previous reviewer pointed you to it.**

_This might help:_ https://javascript.info/promise-error-handling